buildroot-external/package/rpi-eeprom/rpi-eeprom.mk

```diff
@@ -0,0 +1,25 @@
+#############################################################
+#
+# rpi-eeprom
+#
+#############################################################
+RPI_EEPROM_VERSION = f24abcd305ab844dfba2ac92e52883056786ccab
+RPI_EEPROM_SITE = $(call github,raspberrypi,rpi-eeprom,$(RPI_EEPROM_VERSION))
+RPI_EEPROM_LICENSE = BSD-3-Clause
+RPI_EEPROM_LICENSE_FILES = LICENSE
+RPI_EEPROM_INSTALL_IMAGES = YES
+RPI_EEPROM_FIRMWARE_PATH = firmware/stable/pieeprom-2020-09-03.bin
+
+define RPI_EEPROM_BUILD_CMDS
+	$(@D)/rpi-eeprom-config $(@D)/$(RPI_EEPROM_FIRMWARE_PATH) --out $(@D)/default.conf
+	(cat $(@D)/default.conf | grep -v ^$$; echo USB_MSD_PWR_OFF_TIME=1500) > $(@D)/boot.conf
```

Comment:
Here I changed USB_MSD_PWR_OFF_TIME from 1000 to 1500 as workaround for some drives not booting after soft-reboot.

Soon there will be a new beta firmware with a better fix, so we can use this same technique to update to that as part of HassOS update.